source/layouts/layout.erb

```diff
@@ -37,52 +46,105 @@ under the License.
   </head>
 
   <body class="<%= page_classes %>" data-languages="<%=h language_tabs.map{ |lang| lang.is_a?(Hash) ? lang.keys.first : lang }.to_json %>">
-    <a href="#" id="nav-button">
-      <span>
-        NAV
-        <%= image_tag('navbar.png') %>
-      </span>
-    </a>
-    <div class="tocify-wrapper">
+    
+    <div class="ods__documentation-header">
+    
+      <div class="ods__documentation-header-logo">
+        <button type="button" id="nav-button"> Menu </button>
+        <a href="">
+          <img src="../images/ODS_logo api blanc.svg" alt="ods-documentation-logo">
```

Comment:
alt properties should be plain english and should describe the content of the image. No need to say that it's an image. In this case, "OpenDataSoft API documentation" would be a better match.

Also, I'm not convinced by the empty href on the parent anchor tag. It should at least be "/"